### PR TITLE
chore: Add script to list brands from github ⚙️

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

File: scripts/brands.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+const getRepos = limit => githubJsonUrl(`https://api.github.com/orgs/konnectors/repos?per_page=${limit}`)
+const repoFilter = repos => repos
+  .filter(repo => !includes(NOT_KONNNECTORS, repo.name))
+  .map(repo => repo.name)
+const getManifest = repo => githubJsonUrl(`https://raw.githubusercontent.com/konnectors/${repo}/build/manifest.konnector`)
+const getIndex = repo => githubUrl(`https://raw.githubusercontent.com/konnectors/${repo}/build/index.js`)
+
+const getIdentifiers = async repo => {
```
Comment (drazik) [inline scripts/brands.js]: I think adding some comments to this function would help understand what it does. that's not obvious at first sight

File: scripts/brands.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+const githubUrl = url => {
+  return new Promise((resolve, reject) => {
+    request.get(getRequestOptions(url), (error, response, body) => {
+      if (error) {
+        reject(error)
+      }
+      resolve(body)
+    })
+  })
+}
```
Comment (ptbrowne) [inline scripts/brands.js]: request-es6-promise would give you promises with requests automatically

File: scripts/brands.js (hunk 1 above)
Comment (kosssi) [reply scripts/brands.js]: I add more comments ;)

**Comment by kosssi** [comment]
Il faut comprendre pourquoi le build ne fonctionne plus ...

**Comment by ptbrowne** [comment]
Que fait-on de cette PR kosssi ?

**Pull request closed without merging**